src/convenience/keyboard.ts

```diff
@@ -196,6 +196,14 @@ export class Keyboard {
      * end if necessary so you can specify more options in `reply_markup`.
      */
     build() {
+        if (
+            this.selective || this.one_time_keyboard || this.resize_keyboard ||
+            this.input_field_placeholder
+        ) {
+            console.warn(
```

Comment:
Should we use debug instead of console? Otherwise it won't be possible to disable the logging in case this usage is intentional.

Reply:
maybe. can we make it so it's on by default?

Reply:
I don't think so